src/pages/contribute.js

```diff
@@ -1,147 +1,161 @@
+              </TextLink>
+              .
+            </p>
+            <p className="mt-2">
+              For organizations providing products, services, research,
+              networks, or otherwise, use the{" "}
+              <TextLink newWindow href={organizationFormUrl}>
+                Organization Form
+              </TextLink>
+              .
+            </p>
+            <Header className="pt-3">Join our Team</Header>
+            <p className="my-2">
+              Climatescape is maintained by a global team of volunteers. If
+              you&#39;re interested in helping out, here are the ways you can
+              get involved:
+            </p>
+            <ol className="list-decimal list-outside ml-5">
+              <li className="mt-1">
+                Apply to join our{" "}
+                <TextLink newWindow href={contributorFormUrl}>
+                  editorial team
+                </TextLink>
+              </li>
+              <li className="mt-1">
+                Contribute to the development of this website{" "}
+                <TextLink
+                  newWindow
+                  href="https://github.com/bloudermilk/climatescape/issues?q=is%3Aopen+is%3Aissue+label%3A%22good+first+issue%22"
                 >
-                  Contact us
-                </a>
-              </p>
-            </div>
-          </div>
-          <div className="p-3 pb-6 bg-blue-100 lg:p-6">
-            <div className="flex justify-between">
-              <div className="text-gray-900 text-2xl md:text-3xl font-light mb-3">
-                Contributors
-              </div>
-              <button
-                className="lg:hidden"
-                onClick={() => setOpen(!open)}
-                style={{ transform: open ? "none" : "rotateX( 180deg )" }}
-              >
-                <svg
-                  className="fill-current w-10 h-10"
-                  xmlns="http://www.w3.org/2000/svg"
-                  viewBox="0 0 320 512"
-                >
-                  <path d="M31.3 192h257.3c17.8 0 26.7 21.5 14.1 34.1L174.1 354.8c-7.8 7.8-20.5 7.8-28.3 0L17.2 226.1C4.6 213.5 13.5 192 31.3 192z" />
-                </svg>
-              </button>
-            </div>
-            <div className={`${open ? "block" : "hidden"} lg:block`}>
-              {contributors.map(contributor => (
-                <ContributorCard
-                  name={contributor.name}
-                  contributions={contributor.contributions}
-                  avatar={contributor.avatar}
-                  website={contributor.website}
-                />
-              ))}
-            </div>
+                  on GitHub
+                </TextLink>
+              </li>
+            </ol>
+            <p className="my-2">
+              Want to help another way?{" "}
+              <TextLink href="mailto:brendan@sinceresoftware.co">
+                Get in touch
+              </TextLink>
+            </p>
```

Comment:
It would be nice to have another section for partnering with climatescape

this would focus on folks who'd like to either add data or use the data... or partner in other ways, perhaps integrate into the site like climate careers.